tests/h/validators_test.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from mock import Mock
+
+import colander
+
+from h.validators import Email
+
+
+class TestEmail(object):
+    @pytest.mark.parametrize('email', [
+        'jimsmith@foobar.com',
+        'international@xn--domain.com',
+        'jim.smith@gmail.com',
+        'jim.smith@foo.bar.com',
+    ])
+    def test_accepts_valid_addresses(self, schema_node, email):
+        validator = Email()
+        validator(schema_node, email)
+
+    @pytest.mark.parametrize('email', [
+        ' spaces @ spaces.com',
+    ])
```

Comment:
I think we need some more examples of invalid emails, like missing the TLD, or missing the username part, or not containing an "@", etc.

Reply:
Done. Email addresses without TLDs are allowed by the pattern + spec I used, as noted in [this SO answer](https://stackoverflow.com/questions/20573488/why-does-html5-form-validation-allow-emails-without-a-dot).